src/neo4j/neo4j-document.js

```diff
@@ -29,12 +37,33 @@ export async function addDocumentToNeo4j(doc) {
   let arrEdges = [];
   let docElements = doc.elements();
   for (const e of docElements) {
-    if (e.isEntity()) {
+    if (e.isEntity() && !e.isComplex()) {
       let nodeInfo = {
         id: `${e.association().dbPrefix}:${e.association().id}`,
         name: e.association().name
       };
       arrNodes.push(nodeInfo);
+    } else if (e.isEntity && e.isComplex()) {
+      // TODO: Push on the edges that will represent this complex e in arrEdges
+      // Untested
+      const complex = e;
+      const component = makeComponent(complex, doc);
+      for (let i = 0; i < complex.elements.length - 2; i++) {
+        for (let j = i + 1; j < complex.elements().length - 1; j++) {
```

Comment:
Have `.elements` and `.elements()`. Maybe the same?

Reply:
No you're right- that's a careless typo, it should be .elements(), I'll Cmd-F through the file to clean up those mistakes